frontend/src/components/Modal/ProfileModal/ProfileModal.tsx

```diff
@@ -76,6 +76,7 @@ export const ProfileModal = ({ userID, isNew, activeProfile }: ProfileModalProps
                       handleResponse(data, "Editing profile failed!");
                   });
             hideMessage();
+            isVisible();
```

Comment:
question: what does `isVisible` do?

suggestion: rename this prop to something else, since `isVisible` indicates that this prop would be a boolean